src/components/RepeatableField/FieldRow.js

```diff
@@ -204,7 +204,10 @@ class FieldRow extends React.Component {
                         buttonStyle="link"
                         onClick={() => { this.handleRemove(fieldIndex, f); }}
                         aria-label={ariaLabel}
-                        disabled={!canDelete}
+                        disabled={
```

Comment:
@maccabeelevine should we also protect for the cases when `canDelete` is not passed at all?

Reply:
@mkuklis I think that's already taken care of by the defaultProps?

```
FieldRow.defaultProps = {
  canAdd: true,
  canEdit: true,
  canDelete: true,
};
```

Reply:
Ah I missed the default props! This all looks good.